pyOCD/rtos/rtx.py

```diff
@@ -0,0 +1,338 @@
+"""
+ mbed CMSIS-DAP debugger
+ Copyright (c) 2016 ARM Limited
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+from .provider import (TargetThread, ThreadProvider)
+from .common import (read_c_string, HandlerModeThread)
+from ..core.target import Target
+from ..debug.context import DebugContext
+from ..coresight.cortex_m import (CORE_REGISTER, register_name_to_index)
+from pyOCD.pyDAPAccess import DAPAccess
+import logging
+
+# Create a logger for this module.
+log = logging.getLogger("RTX")
```

Comment:
I'm not sure it matters (are logger names case-sensitive?), but all loggers in pyOCD are lower case.